lisa/sut_orchestrator/azure/common.py

```diff
@@ -190,6 +190,7 @@ class AzureNodeSchema:
     # It decides the real computer name. It cannot be too long.
     short_name: str = ""
     vm_size: str = ""
+    excluded_vm_sizes: Optional[List[str]] = None
```

Comment:
When is it needed? Can it be used by some feature to exclude related vm sizes?

Reply:
in our testing, there are some internal sizes, they have quota, but we may see provision time out issue or they don't offer the capabilities which they should have.